ballerina/types.bal

```diff
@@ -56,3 +56,25 @@ public type CorsConfig record {|
 isolated service class HttpService {
     *http:Service;
 }
+
+// GraphQL client related data binding types representation
+
+# Represents the target type binding record with data and extensions of a GraphQL response for `executeWithType` method.
+#
+# + extensions -  Meta information of the GraphQL API call
+# + data -  Data information of the GraphQL API call
+public type GenericResponse record {|
+   map<json?> extensions?;
+   record {| anydata...; |}|map<json?> data?;
+|};
+
+# Represents the target type binding record with data, extensions and errors of a GraphQL response for `execute` method.
+#
+# + extensions - Meta information of the GraphQL API call  
+# + data - Data information of the GraphQL API call
+# + errors - Error information of the GraphQL API call
```

Comment:
```suggestion
# + errors - The errors occurred (if present) while processing the GraphQL request.
```

Reply:
Replaced in [f26678c](https://github.com/ballerina-platform/module-ballerina-graphql/pull/663/commits/f26678c4ceef61a6ce293851142e167fd6e26c07). Thank you.